package.json

```diff
@@ -82,6 +83,7 @@
     "electron-builder": "^23.1.0",
     "electron-devtools-installer": "^3.2.0",
     "electronmon": "^2.0.2",
+    "node-loader": "^2.0.0",
```

Comment:
Where is it used?

Reply:
see https://github.com/zhxie/anioke/pull/79/files#diff-403f360037af794dde0b529c0c12fbb335b39c8e952fb3ba36029228fe5838ba